cabal.project

```diff
@@ -1,5 +1,9 @@
 packages:
   ./.
-  ./plutarch-benchmark/plutarch-benchmark.cabal
+  ./plutarch-*
```

Comment:
I don't understand cabal properly yet; however wildcards always make me nervous.

Are there any implications here for future additions to 'plutarch' causing unexpected behaviour?

Reply:
Using wildcards in cabal.project is not uncommon. In fact the default value already uses one. See https://cabal.readthedocs.io/en/3.6/cabal-project.html#cfg-field-packages